server/container/guice/memory/src/main/java/org/apache/james/modules/mailbox/MemoryMailboxModule.java

```diff
@@ -99,6 +97,7 @@ protected void configure() {
         install(new MemoryDeadLetterModule());
         install(new MemoryQuotaModule());
         install(new MemoryQuotaSearchModule());
+        install(new LuceneSearchMailboxModule());
```

Comment:
Not sure we want to use the File Directory (in `LuceneSearchMailboxModule`) for the memory app.

Memory app leaving some files on the host file system after testing sounds weird, no?

Likely we should use the Ram directory instead cf `LuceneMemorySearchMailboxModule`?

Reply:
another reason: RAM is way faster than disk for testing. 

AFAIK we do use memory directory for Lucene is tests too.